### DISCUSSION TITLE
Possible to add multiple themes aside from dark and light?

### DISCUSSION THREAD

**Discussion by LeunensMichiel** [mantinedev/mantine#1300] [Q&A] [answered]
We want to implement something similar to `Slack`'s and `Github`'s theming choices. They have 3 to 6 light themes and 3 to 6 dark themes. Is there a way to override the default `dark` and `light` color schemes?

**Comment by rtivital** [comment, marked as answer]
You can change `theme.white`, `theme.black` and `theme.colors` to change colors any way you want

**Reply by LeunensMichiel** [reply]
Ah, stupid of me. Thank you for the quick response!

**Reply by LeunensMichiel** [reply]
@rtivital my one gripe with Mantine was that I couldn't set the default color for my swatches (eg: force brand color for buttons). As of `4.2.0` this is now fixed and I couldn't be happier. 

Your library rocks! Thank you for your hard work. 🎉

**Reply by rtivital** [reply]
Thanks for kind words!
